src/rf/js/src/home/controllers.js

```diff
@@ -7,19 +7,39 @@ var $ = require('jquery'),
     settings = require('../settings'),
     Library = require('./components/library'),
     Modals = require('./components/modals'),
-    uploads = require('../core/uploads'),
     models = require('../core/models');
 
+function setupPendingLayers() {
+    var pendingLayers = new models.PendingLayers(),
+        pollInterval = 1000;
```

Comment:
Consider increasing this since the geoprocessing will take 10-15 minutes to complete.